### DISCUSSION TITLE
Show state of different entity and not the graph

### DISCUSSION THREAD

**Discussion by EmilonGitHub** [kalkih/mini-graph-card#725] [Q&A]
Hi guys,

I have my tablets battery life displayed like this:
![image](https://user-images.githubusercontent.com/95291703/153278001-f7080d92-507d-428a-a128-70a80d76cb17.png)
For that I use the battery_level sensor:
![image](https://user-images.githubusercontent.com/95291703/153278083-949b4a55-a921-44b4-a142-0253c8d93cc7.png)
I want to show somehow, that my device is charging when it's charging. For that, I need to use a different entity, but I do not want to have that entity included in my graph. I want to change the color of the graph or the Icon if my device is charging. 
Is this possible?
Emil

**Comment by MSZ-Spyro** [comment]
Add 
```
- entity: whatever
  show_graph: false
  show_state: true
```
under the second entity.
